docs/manual/src/interface_definition.md

```diff
@@ -0,0 +1,80 @@
+# The Interface Definition
+
+The public interface of your Rust crate should be defined as an inline module
+decorated with the `#[uniffi_macros::declare_interface]` macro. Code inside
+this macro is restricted to a small subset of the Rust language that UniFFI
+understands and that can be safely exposed to other languages.
+
+Here is a small but relatively complete example:
+
+```rs
+//!
+//! This is an example Rust crate to be exposed via UniFFI.
+//! It's a subset of our "sprites" example crate.
+//!
+
+// It can use things from stdlib, submodules, or other Rust crates.
+use std::io::prelude::*;
+
+// But its public interface should be defined inside the `declare_interface` macro.
+#[uniffi_macros::declare_interface]
+mod sprites {
+
+  // The interface can define "records" as structs with public fields.
```

Comment:
I don't think this comment is as clear as it could be either - it says `"records"` but doesn't state what records are. Saying "they do not have any public method impls" contradicts below where one does.

Maybe you were trying to say something like "A simple struct without an public method impls can be considered a kind of simple 'record'" or similar?